app/components/collections/link_to_show_component.html.erb

```diff
@@ -0,0 +1 @@
+<td class="collection-title"><%= link %></td>
```

Comment:
I'm not terribly fond of having the `<td>` in this component, as that limits where we're able to use it.  Can that be moved back into ApprovalsComponent?